src/main/webapp/index.html

```diff
@@ -0,0 +1,122 @@
+        alt="Toba Olokungbemi" width="300" height="300">
+        <figcaption>Toba is currently a Computer Science student in Harvard
+            University's Class of 2022. He intends to become a Software Engineer 
+            after graduating!</figcaption>
+      </figure> -->
+      
+      <br>
+      <h2>Our Reasoning</h2>
+      <br>
+      <p> The college application process is one of the most stressful
+          periods during a teen’s academic career.  Amongst difficult 
+          course work, standardized testing, multiple questions probing
+          into their qualifications and life story, high schoolers are
+          constantly being reminded that the decisions made during this
+          time will determine the course of the rest of their lives.
+          Consequently, the pressure to find the “perfect college” drives
+          students to scour the internet searching for videos, articles,
+          and reviews that could provide any insight into what their life
+          would look like on a particular campus. Sites like Niche.com or
+          StudentReviews.com offer students this by allowing students that
+          actually attend the school to rate and review their experiences. 
+      </p>
+      <br>
+      <p>As once prospective high school applicants and now college
+          students, one thing the creators noticed is that it is very difficult for
+          minority students to find insightful information about the cultural
+          communities on these websites. Especially in predominantly white
+          institutions, the minority communities often make up just a small
+          percentage of the school. Despite being a fraction of the overall
+          population, they often have very strong, active communities to the
+          point that the experience of a minority student will be vastly
+          different than that of their white counterparts. Because these
+          communities only make up a small percentage of the school, these
+          experiences are not highly visible on these college rating sites.
+          The sentiments expressed on these websites do not necessarily provide
+          minority prospective students that foresight into what their own
+          lives will be like.
+      </p>
+      <br>
+    
+    </div>
+<!-- Second Search Bar (POST MVP)     
+    <div id="search bar">
+      <input type="text" id="school-search" onkeypress="hideElements()"/>
+      <home-button onclick="loadSchoolInfo()">Search</home-button>
+    </div> -->
+    <main id="main">
+      <h1 id="school-name"></h1>
+
+      <section>
+        <p id="school-desc"></p>
+        <p id="students"></p>
+      </section>
+
+      <section>
+        <p id="admissions"></p>
+        <p id="cost"></p>
+      </section>
+
+      <section>
+        <p id="race-piechart"></p>
+        <p id="gender-piechart"></p>
+      </section>
+      <a class="reviews-button" href="comments.html">Student Reviews</a>
```

Comment:
You should be able to remove this since it will be on the college-info.html page.

Reply:
I can't remove this because the way the site retrieves data from the API, there wouldn't be a way to make the search bar on the home page functional (to my knowledge at least). It takes an input from the *current* html page, and if I redirect from the homepage to the college-info page onsearch, the current input goes from whatever the user typed to an empty string. I don't know of a way to get around this.